packages/ETLoad.package/ETLPlaceholderObject.class/class/compatibilityTests.st

```diff
@@ -0,0 +1,23 @@
+compatibility
+compatibilityTests
```

Comment:
Please add a comment explaining the dictionary structure (key: squeak 3 object format, value: block testing if a squeak 5 object format is compatible to the squeak 3 format).